rspec-mode.el

```diff
@@ -225,6 +225,13 @@ info, are considered errors."
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-holder-dirs '("app" "lib")
```

Comment:
`applike` and `container` both sound fine for me, although the latter is very close to `holder`.